content/languages.yml

```diff
@@ -0,0 +1,124 @@
+- name: Arabic
+  translated_name: العربية
+  code: ar
+  is_completed: false
+- name: Azerbaijani
+  translated_name: Azərbaycanca
+  code: az
+  is_completed: false
+- name: Bulgarian
+  translated_name: Български
+  code: bg
+  is_completed: false
+- name: Bengali
+  translated_name: বাংলা
+  code: bn
+  is_completed: false
+- name: German
+  translated_name: Deutsch
+  code: de
+  is_completed: false
+- name: Spanish
+  translated_name: Español
+  code: es
+  is_completed: true
+- name: Persian
+  translated_name: فارسی
+  code: fa
+  is_completed: false
+- name: French
+  translated_name: Français
+  code: fr
+  is_completed: false
+- name: Hebrew
+  translated_name: עברית
+  code: he
+  is_completed: false
+- name: Hindi
+  translated_name: हिन्दी
+  code: hi
+  is_completed: false
+- name: Armenian
+  translated_name: Հայերեն
+  code: hy
+  is_completed: false
+- name: Indonesian
+  translated_name: Bahasa Indonesia
+  code: id
+  is_completed: false
+- name: Italian
+  translated_name: Italiano
+  code: it
+  is_completed: false
+- name: Japanese
+  translated_name: 日本語
+  code: ja
```

Comment:
Technically Japanese and Brazillian Portuguese aren’t *completely ready yet so I’d change these to false for now.

Reply:
What's the threshold? I don't think they need to be "completely ready".

I guess we can bucket things into three groups:
* Completed (95%+): Show language subdomain and "contribute" links
* Partly translated (25%-95%): Show language subdomain and "contribute" links
* In-progress (< 25%): Show "contribute" links

The question is what should those thresholds be?

Reply:
The “100%” is for a “core” subset of pages defined by Dan (home page, tutorial, basics, apis). For example, some of the Spanish hooks docs aren’t done yet but it”s still 100% since it finished the core. Japanese and Portuguese are at around 70% on those (Jp doesn’t have the tutorial yet). Plus, there are final tasks like checking for consistency of terminology that need to be done before marking as complete (which I’m going through with Spanish right now)

Reply:
That's a little more information than I had before, but I think it's still a bit vague to me. Is this written down somewhere in a more objective format, that we can refer to if we need to know which bucket a given subdomain falls into?

Reply:
I picked some somewhat arbitrary % thresholds for now (specified via inline comments).

Reply:
Okay! I’ll make a more concrete version of it sometime today

Reply:
I made a gist here: https://gist.github.com/tesseralis/2e57ab7605f844ec8523284c6b70e73a

The milestones are based on my perception of how the current translation efforts are going. Please let me know if this makes sense.

Reply:
So in this case:

Released: Spanish

Pre-release 🎁: Japanese, Portuguese (BR)

In Progress: Arabic, Azerbaijani, Bulgarian, etc.

Reply:
@gaearon what do you think?

Reply:
At first look, I think the proposed categories are maybe too fine grained.

I don't think it's necessary (or all that helpful) to distinguish between released, complete, and complete+. I also don't really want to have to move sites around between those categories each time we make significant changes to core pages.

Similarly, I'm not sure there's much value in distinguishing between "in-progress" and lesser translated states. I think it's worth showing _all_ translations so that we direct potential contributors to them. I think it _is_ important to indicate "this is not done yet" so we don't get people's hopes up for nothing– but I think the current 3 enums already do this.

Reply:
@bvaughn It wouldn't be different sections for "complete"/"complete+" stuff and we don't have to add it in now. It's morel like... giving maintainers a gold star to encourage them to go beyond the core pages of translations.

In that case I'd rather have just "released" and "in progress", without an arbitrary cut-off point. That might be the best option.

But I also want @gaearon to weigh in.

Reply:
Seems like you two agree on the number of sections on the page but @tesseralis wants to add extra “distinguishing” marks to some translations? I’d say it’s fine to leave it out of the MVP and add a bit later.

Reply:
Fine by me. I added the extra steps to isreactreadyyet.com since it's easier to verify those :)